cmd/release-controller/controller.go

```diff
@@ -485,6 +506,33 @@ func (c *Controller) processNextAudit() bool {
 	return true
 }
 
+func (c *Controller) bugzillaWorker() {
+	for c.processNextBugzilla() {
+	}
+	glog.V(4).Infof("Worker stopped")
+}
+
+func (c *Controller) processNextBugzilla() bool {
+	obj, quit := c.bugzillaQueue.Get()
+	if quit {
+		return false
+	}
+	defer c.bugzillaQueue.Done(obj)
+
+	// don't run if we don't have a verifier
+	if c.bugzillaVerifier != nil {
+		return true
+	}
+	key := obj.(queueKey)
+
+	glog.V(5).Infof("processing %v begin", key)
+	err := c.syncBugzilla(key)
+	c.handleNamespaceErr(c.bugzillaQueue, err, key)
```

Comment:
This is the logic that allows you to exit early with error (on any "retriable error") in syncBugzilla.  In general, all errors will be retriable because there is no reason your code should fail to get and update bugs, and if it does, we will have to observe that in the logs and fix bugs in the code.